tool/grind.dart

```diff
@@ -54,6 +54,7 @@ void main(List<String> args) {
     'info',
     'render',
     'renderSync',
+    'Exception',
```

Comment:
@jerivas I assume it's okay to expose this? The JS API Spec tests were using it.